Source/WebCore/html/HTMLDialogElement.cpp

```diff
@@ -77,11 +77,11 @@ ExceptionOr<void> HTMLDialogElement::show()
 
     setBooleanAttribute(openAttr, true);
 
-    m_previouslyFocusedElement = document().focusedElement();
+    m_previouslyFocusedElement = protectedDocument()->focusedElement();
```

Comment:
Instead of calling protectedDocument() twice in row.
Can we allocate a `RefPtr` local variable and store document there?